pkg/master/master.go

```diff
@@ -790,10 +832,21 @@ func (m *Master) thirdpartyapi(group, kind, version, pluralResource string) *api
 
 	apiRoot := makeThirdPartyPath("")
 
+	thirdPartyAPIPath := makeThirdPartyPath(group)
+
 	storage := map[string]rest.Storage{
 		pluralResource: resourceStorage,
 	}
 
+	// We can have more than one thirdpartyresourcekind for an APIGroup (thirdpartyapi)
+	// /apis/domain.com/rest1 and /apis/domain.com/rest2
```

Comment:
not fixed? /apis/domain.com/v1/rest2

/apis/domain.com/v1/rest1 and /apis/domain.com/v1/rest2

or /apis/domain.com/v1/rest1 and  /apis/domain.com/v2/rest1

Illustrating different resources or different versions.